config.template.js

```diff
@@ -9,5 +9,6 @@ window.env = {
   CMS_API_HOST: '$CMS_API_HOST',
   ORGANIZATION_HOST: '$ORGANIZATION_HOST',
   ORGANIZATION_CATALOGUE_HOST: '$ORGANIZATION_CATALOGUE_HOST',
-  REPORT_API_HOST: '$REPORT_API_HOST'
+  REPORT_API_HOST: '$REPORT_API_HOST',
+  METADATA_QUALITY_ASSESSMENT_API_HOST: '$METADATA_QUALITY_ASSESSMENT_API_HOST'
```

Comment:
Ta opp tråden med @valosnah for å legge inn denne miljøvariabelen

Reply:
Fix'd